### PR TITLE
Allow compatibility with Symfony 2.3

### PR DESCRIPTION
**Pull request by stefk**
This is just a version constraint update.

### CODE REVIEW

**Comment by stefk** [comment]
Well, I just realized a similar pull request has been submitted 2 weeks ago, making this one totally useless :/

**Comment by stefk** [comment]
Edit : I think there's an issue in the pull request mentionned above, so I just reopen this one, hoping it will be merged soon :)

**Comment by jonathaningram** [comment]
@schmittjoh what do you think about merging this?

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=5.3.2",
         "jms/twig-js": "dev-master",
-        "symfony/framework-bundle": "2.*,<2.3"
+        "symfony/framework-bundle": "2.*,<2.4"
```
Comment (schmittjoh) [inline composer.json:17]: Can you change this to `~2.0`?

Comment (schmittjoh) [inline composer.json:17]: nvm, I made the change :)

**Pull request closed without merging**